### ISSUE TITLE
Chapter 4, 4th edition method deprecation

### ISSUE THREAD

**Issue by chessboards** [head-first-csharp/fourth-edition#19]
The method used on page 160 for checking whether or not a float is infinite looks to have been replaced by `float.IsInfinity()`, at least in .NET version 5.0.300
![image](https://user-images.githubusercontent.com/64972947/124513119-1286cb00-dda8-11eb-8d83-512fc2cb1c5f.png)

Negating the result of IsInfinity() would fix the example of any errors

**Comment by andrewstellman** [comment]
Good catch—thanks! You're right, that bit of code was never changed from the 3rd edition of the book, before IsFinite was deprecated. I'll make sure future printings and digital editions are updated.
